src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -425,6 +425,23 @@ export class RoslynLanguageServer {
                 if (solutionUris) {
                     if (solutionUris.length === 1) {
                         this.openSolution(solutionUris[0]);
+                    } else if (solutionUris.length > 1) {
+                        // We have more than one solution, so we'll prompt the user to use the picker.
+                        const chosen = await vscode.window.showInformationMessage(
+                            vscode.l10n.t(
+                                'Your workspace has multiple Visual Studio Solution files; please select one to get full IntelliSense.'
+                            ),
+                            { title: vscode.l10n.t('Choose solution'), open: true },
+                            { title: vscode.l10n.t('Do not load solutions'), open: false }
+                        );
+
+                        if (chosen) {
+                            if (chosen.open) {
+                                vscode.commands.executeCommand('dotnet.openSolution');
```

Comment:
Do we also want to update the property or no like Dev Kit? We've been seeing feedback where people do have multiple solutions and might be confused if this does change it.

Reply:
@dibarbet had the idea of just adding a third button. With some wordsmithing I think I got them all to fit.